src/test/lib/concurrency/stress_test.cpp

```diff
@@ -524,21 +524,17 @@ TEST_F(StressTest, ConcurrentInsertsSetChunksImmutable) {
   }
 
   // 100 threads * 300 insertions * 2 values = 60'000 tuples in 20'000 chunks with chunk size 3.
```

Comment:
Do you think the previous set up had a value? We could have 301 insertions and thus an incomplete mutable chunk again!?!?